src/fanuc_sim/description/scene.xml

```diff
@@ -3,6 +3,7 @@
   <compiler angle="radian" autolimits="true" />
   <option integrator="RK4" timestep="0.005" />
   <include file="lrmate200id/lrmate200id_globals.xml" />
+  <include file="tool.xml" />
```

Comment:
`lrmate200id_globals.xml` needs some KP and other tuning. I'll pull some in from https://github.com/PickNikRobotics/picknik_accessories/tree/pr-fanuc-workshop-assets

Right now it is overshooting and hitting the tool during docking very aggressively.